backend/legacyapi/project.go

```diff
@@ -22,6 +22,8 @@ const (
 	// This can be mutated by the user. But for now this is only used by onboarding flow to create
 	// a prod instance after first signup, so it's safe to refer it.
 	DefaultProdEnvironmentID = "prod"
+	// DefaultProdEnvironmentUID is the initial resource UID for the prod environment.
+	DefaultProdEnvironmentUID = 102
```

Comment:
It's a duplicate message for `DefaultProdEnvironmentID`, I'd like to use the `DefaultProdEnvironmentID` to find the environment.

Reply:
This is more involved than originally thought, will spawn another thread to tackle this